Neos.Neos/Classes/Fusion/Helper/SiteHelper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Fusion\Helper;
+
+use Neos\ContentRepository\Core\Projection\ContentGraph\NodePath;
+use Neos\Neos\Domain\Model\Site;
+use Neos\Neos\Domain\Model\SiteNodeName;
+use Neos\Neos\Domain\Repository\SiteRepository;
+use Neos\Neos\FrontendRouting\NodeAddressFactory;
+use Neos\ContentRepository\Core\Projection\ContentGraph\Node;
+use Neos\ContentRepositoryRegistry\ContentRepositoryRegistry;
+use Neos\Flow\Annotations as Flow;
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Neos\Domain\Exception;
+
+/**
+ * Eel helper for accessing the Site object
```

Comment:
maybe add an @internal comment saying this is mostly relevant for legacy (pre 8.3) migration? I believe this shouldn't be used in newer instances.

Reply:
In Neos.Seo we have a similar use case: https://github.com/neos/neos-seo/blob/c047aeab7b63e860cedc33c45a36a2a1a2e28490/Resources/Private/Fusion/Metadata/TitleTag.fusion#L2

Reply:
@ahaeslich do you think the helper and method name is OK, or do you have other suggestions?

Reply:
I'm fine with both. 

What is perhaps more confusing now that we have this helper is that we use `site` in fusion instead of `siteNode` and I wonder if we should change that. 🤔 

But preferably with another MR to keep this one simple 😉.

Reply:
@ahaeslich hm, I would not change it, because it introduces a rather big backwards incompatible change (which I try to avoid whereever possible/sensible)... I agree if we started from scratch I'd change the naming; but right now everybody is accustomed to "site" I guess.

Reply:
Sorry, commented below on the whole PR, but I think we should change it, especially now. It seems so confusing to me the further we split the worlds apart.